lib/hako/container.rb

```diff
@@ -212,6 +213,7 @@ def default_config
         'port_mappings' => [],
         'volumes_from' => [],
         'privileged' => false,
+        'docker_security_options' => [],
```

Comment:
This should also be `nil` ?

Reply:
7c101da 🔨 

(Other optional array options (e.g. `links`) are set to `[]` by default and not allowed to be set to null, so I had treated `docker_security_options` in the same way.